rgw/v2/lib/s3/lifecycle_ops.py

```diff
@@ -0,0 +1,49 @@
+import os, sys, glob
```

Comment:
Can you change the filename? there is already a filename called test_lifecycle_ops.py or similar one. this is very similar to that. it's little ambiguous.

Reply:
Done, I have re-named the file to ../../lib/s3/lifecycle_validation.py